spec/requests/api/drops_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+
+RSpec.describe Api::DropsController, :vcr => {:cassette_name => "place" }, type: :request do
```

Comment:
According to me the best name is no name. :-)
If you label the test like this: `RSpec.describe Api::DropsController, :vcr, type: :request do` VCR will automagically create cassette in `spec/fixtures/[describe_block_name]/[test_block_name]`.